### PR TITLE
Add aarch64-linux-unknown-musl builds for nimbus-fml and nimbus-cli [ci full]

### PR DESCRIPTION
**Pull request by brennie**
### Pull Request checklist ###

- **Breaking changes**:  This PR follows our [breaking change policy](https://github.com/mozilla/application-services/blob/main/docs/howtos/breaking-changes.md)
  - [X] This PR follows the breaking change policy:
     - This PR has no breaking API changes
- [ ] **Quality**: This PR builds and tests run cleanly
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
  - only adds build target
- [X] **Changelog**: This PR includes a changelog entry in [CHANGELOG.md](../CHANGELOG.md) or an explanation of why it does not need one
  - only adds build target
- [X] **Dependencies**: This PR follows our [dependency management guidelines](https://github.com/mozilla/application-services/blob/main/docs/dependency-management.md)
  - No new deps

[Branch builds](https://github.com/mozilla/application-services/blob/main/docs/howtos/branch-builds.md): add `[firefox-android: branch-name]` to the PR title.

### CODE REVIEW

File: taskcluster/app_services_taskgraph/transforms/nimbus.py [hunk 1]
```diff
@@ -8,14 +8,26 @@
...
+LINUX_BUILD_TARGETS = (
+    'aarch64-unknown-linux-musl',
+    'x86_64-unknown-linux-gnu',
+    'x86_64-unknown-linux-musl',
+    'x86_64-pc-windows-gnu',
+)
+
+MAC_BUILD_TARGETS = (
+    'x86_64-apple-darwin',
+    'aarch64-apple-darwin',
```
Comment (jhugman) [inline taskcluster/app_services_taskgraph/transforms/nimbus.py:20]: Nice.

**Review by jhugman — DISMISSED** [review]
This is exactly what I was hoping it would be. Thanks!

**Comment by brennie** [comment]
Last push was just a rebase.

**Review by jhugman — DISMISSED** [review]
r+ ✅ 🚢 :shipit:

**Comment by brennie** [comment]
Adding musl builds requires either building our own cross compiler or waiting for support for aarch64 builders in TC.

**Pull request closed without merging**